### PR TITLE
MeasureConfigHelper

### PR DESCRIPTION
**Pull request by saquino0827**
### Information
- MeasureConfigHelper to extract the measure configuration from a node.

### Changes proposed in this PR:
- Add new helper for measure configurations.
- Refactor validations to use the MeasureConfigHelper.
- Refactor MipsQualityMeasureValidator to remove additional configuration retrieval.

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [x] New unit tests written to cover new functionality.
- [x] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [x] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/main/java/gov/cms/qpp/conversion/util/MeasureConfigHelper.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		// private for this helper class
+	}
+
+	/**
+	 * Convenience method to retrieve the measure configuration for validation from an ecqm node
+	 *
+	 * @param node Contains the id that associates with the measure config
+	 * @return
+	 */
+	public static MeasureConfig getMeasureConfig(Node node) {
```
Comment (halprin) [inline converter/src/main/java/gov/cms/qpp/conversion/util/MeasureConfigHelper.java]: What was your thoughts on rolling this in to `MeasureConfigs` (notice the `s`) class?
Granted, I don't think I would want `MeasureConfigs` to know about `Node`s...

Comment (saquino0827) [reply converter/src/main/java/gov/cms/qpp/conversion/util/MeasureConfigHelper.java]: Initially I thought about that, however, I liked the idea of passing in a node to reduce some code duplication. I also didn't want `MeasureConfigs` to know about the `Node` class.

File: converter/src/main/java/gov/cms/qpp/conversion/validate/MipsQualityMeasureIdValidator.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import gov.cms.qpp.conversion.model.validation.MeasureConfigs;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/MipsQualityMeasureIdValidator.java]: MINOR Remove this unused import 'gov.cms.qpp.conversion.model.validation.MeasureConfigs'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [MipsQualityMeasureIdValidator.java#L20](https://github.com/CMSgov/qpp-conversion-tool/blob/7dbb6ec94d35326b89f16f063c4dc7eb60798cb4/converter/src/main/java/gov/cms/qpp/conversion/validate/MipsQualityMeasureIdValidator.java#L20): Remove this unused import 'java.util.Map'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**